docs/source/installation.rst

```diff
@@ -0,0 +1,50 @@
+.. _installation:
+
+Installation
+============
+It is highly recommended to install the project into a new virtual environment.
+
+
+#Installation from PyPI
+#----------------------
+#The :code:`atldld` package can be easily installed from PyPI.
+#
+#
+#.. code-block:: bash
+#
+#    pip install atldld
```

Comment:
I don't think this kind of comments (with a `#`) work in rst-files.

I just check how to do that, and it seems commenting out lines in rst is awefule. One way that seems to work is this:
```rst
..
    Installation from PyPI
    ----------------------
    The :code:`atldld` package can be easily installed from PyPI.


    .. code-block:: bash

        pip install atldld
```
But maybe if we're going to upload this package to PyPI soon anyway we can leave this block in already.